packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -36,6 +49,21 @@ initializeFluentUiIcons();
  */
 export type ContainerSummaryViewProps = HasContainerId;
 
+const columnsDef: TableColumnDefinition<Item>[] = [
+	createTableColumn<Item>({
+		columnId: "containerProperty",
+		renderHeaderCell: () => <>Boi</>,
+	}),
+	createTableColumn<Item>({
+		columnId: "value",
+		renderHeaderCell: () => <>gyal</>,
```

Comment:
Not sure how these are used, just making a note in case this renders a table with boi and gyal somewhere